nevergrad/functions/powersystems/test_core.py

```diff
@@ -12,3 +13,13 @@ def test_powersystem() -> None:
     x = 7 * np.random.rand(func.dimension)
     value = func.function(x)  # should not touch boundaries, so value should be < np.inf
     assert value < np.inf
+
+@patch("%s.core.plt" % __name__)
```

Comment:
can you use [fstrings](https://www.python.org/dev/peps/pep-0498/) instead, those are easier to read ;)

Reply:
Sounds good! Just changed to f strings and pushed.